documentation/source/user_section/tutorials/lesion-analysis/lesion-statistics/lesion-metrics.rst

```diff
@@ -0,0 +1,56 @@
+Compute lesion metrics using ``sct_analyze_lesion``
+###################################################
+
+The ``sct_analyze_lesion`` function computes various morphometric measures on segmented lesions and spinal cord. The function assigns an ID value to each lesion (1, 2, 3, etc.) and then outputs morphometric measures for each lesion to a XLS file.
+
+The following morphometric measures are computed:
+
+* ``volume [mm^3]`` : volume of the lesion
+* ``length [mm]`` : length of the lesion along the superior-inferior (SI) axis
+
+.. figure:: https://raw.githubusercontent.com/spinalcordtoolbox/doc-figures/master/lesion-analysis/intramedullary-lesion-length.png
+  :align: center
+  :figwidth: 60%
+
+* ``max_equivalent_diameter [mm]`` : maximum diameter of the lesion, when approximating the lesion as a circle in the axial plane
+* ``max_axial_damage_ratio []`` : maximum axial damage ratio defined as the ratio of the lesion area divided by the spinal cord area. The ratio is computed in the axial plane for each slice and and the maximum ratio is reported.
+
+.. figure:: https://raw.githubusercontent.com/spinalcordtoolbox/doc-figures/master/lesion-analysis/axial-damage-ratio.png
+  :align: center
+  :figwidth: 60%
+
+TODO: Ask Andrew Smith for permission to use the figure
+
+* ``dorsal_bridge_width [mm]`` : dorsal tissue bridges defined as the width of dorsal spared tissue (i.e. towards the posterior direction of the AP axis) at the minimum distance from the intramedullary lesion edge to the boundary between the spinal cord and cerebrospinal fluid
+* ``ventral_bridge_width [mm]`` : ventral tissue bridges defined as the width of ventral spared tissue (i.e. towards the anterior direction of the AP axis) at the minimum distance from the intramedullary lesion edge to the boundary between the spinal cord and cerebrospinal fluid
+
+.. figure:: https://raw.githubusercontent.com/spinalcordtoolbox/doc-figures/master/lesion-analysis/tissue-bridges.png
+  :align: center
+  :figwidth: 80%
+
+Running ``sct_analyze_lesion``
+------------------------------
+
+``sct_analyze_lesion`` can be applied to lesion and spinal cord segmentation masks using the following command
+
+.. code:: sh
+
+   sct_analyze_lesion -m t2_lesion_seg.nii.gz -s t2_sc_seg.nii.gz
+
+:Input arguments:
+   - ``-m`` : 3D binary mask of the segmented lesion
+   - ``-s`` : 3D binary mask of the segmented spinal cord
```

Comment:
The tutorial currently does not cover the `-f` input arg (atlas/template) ([L74-L76](https://github.com/spinalcordtoolbox/spinalcordtoolbox/blob/master/spinalcordtoolbox/scripts/sct_analyze_lesion.py#L74-L76)). Maybe we could also add a section about this arg. Possibly as a new page so that the current page is not too heavy?

Reply:
We agreed during the 2024-08-08 SCT dev meeting that adding a tutorial for the `-f` arg is a good idea!

Reply:
Done in https://github.com/spinalcordtoolbox/spinalcordtoolbox/pull/4586/commits/79e9f0b9881daa66331486010d2756a116ea0ed8